configuration/purging.md

```diff
@@ -45,13 +45,14 @@ To enable purging, create a purge function and write it into `app_settings.json`
 ```json
     "//": "other app_settings settings",
     "purge": {
-        "fn": "function(contact, reports) { /*your logic goes here*/ }"
+        "fn": "function(userCtx, contact, reports) { /*your logic goes here*/ }"
     }
 ```
 
-This function takes two parameters:
- - `contact`, the contact document of a patient or other contact who has reports about them
- - `reports`, an array of all reports for that patient that are present on the device (if you have already purged a report it will not show up here)
+This function takes three parameters:
+ - `userCtx`, an object with the user's `name` and `roles` as fields, which is particularly useful to configure different purging functions for different roles.
```

Comment:
Can you either link to (if one already exists in docs somewhere) or give a structural example of what this looks like.

Reply:
Sure can!